docs/pom.xml

```diff
@@ -100,6 +105,19 @@
             <artifactId>quarkus-langchain4j-anthropic-deployment</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.quarkiverse.langchain4j</groupId>
```

Comment:
@geoand I wasn't sure if I needed to do this in order for the easy-rag configuration adocs to get generated?

Reply:
+1